db/models.py

```diff
@@ -40,8 +47,59 @@ def __str__(self):
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.CASCADE)
+    cinema_hall = models.ForeignKey(CinemaHall,
+                                    on_delete=models.CASCADE)
     movie = models.ForeignKey(to=Movie, on_delete=models.CASCADE)
 
     def __str__(self):
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
+
+    def __str__(self):
+        return str(self.created_at)
+
```

Comment:
Order's QuerySet should be ordered from newest to oldest by default.

Reply:
Thanx I fixed it)